src/App.js

```diff
@@ -1,29 +1,69 @@
 import './App.css';
+import '../node_modules/bootstrap-icons/font/bootstrap-icons.css'
+import {useState} from 'react'
+import {render} from 'react-dom'
+let list=[];
+const input=document.getElementById('input');
```

Comment:
this is an unnecessary line, in React components we don't explicitly need to catch the `input` field like that.
just create any input tag in your return and React will do that for you behind the scene. for more check [here](https://reactjs.org/docs/rendering-elements.html) and [this one](https://www.freecodecamp.org/news/jsx-in-react-introduction/)